schutzbot/slack_notification.sh

```diff
@@ -2,7 +2,7 @@
 
 set -eux
 
-MESSAGE="\"Load testing on image builder finished with status *$1* $2 \n QE: @jabia \n Link to results: $CI_PIPELINE_URL \""
+MESSAGE="\"Load testing on image builder finished with status *$1* $2 \n <$CI_PIPELINE_URL|Results> - QE: @atodorov, @jrusz, @jabia \""
```

Comment:
Hm. don't you think it would be better to just put the link under the main text? Just to make it shorter, we don't need another line that just says "Results"

```suggestion
MESSAGE="\"<$CI_PIPELINE_URL|Load testing on image builder> finished with status *$1* $2 \n QE: @atodorov, @jrusz, @jabia \""
```

Reply:
I actually removed one of the lines. But yeah, that would look much better!